Foundation/NSURLSession/http/HTTPURLProtocol.swift

```diff
@@ -127,6 +129,7 @@ fileprivate extension _HTTPURLProtocol {
                 set(requestBodyLength: .noBody)
             case (_, .some(let length)):
                 set(requestBodyLength: .length(length))
+                bodyLength = Int64(length)
```

Comment:
what happens if an attacker sends a content-length that exceeds `Int64` but not `UInt64`? Could we have a test case for that? I already added a few test cases for a misbehaving server, this one should be really easy to add.